README.md

```diff
@@ -71,44 +71,50 @@ The master branch works with **PyTorch 1.5+**.
 
 </details>
 
-Apart from MMDetection, we also released a library [mmcv](https://github.com/open-mmlab/mmcv) for computer vision research, which is heavily depended on by this toolbox.
+Apart from MMDetection, we also released [MMCV](https://github.com/open-mmlab/mmcv) for computer vision research and [MMEngine](https://github.com/open-mmlab/mmengine) for model training, that are heavily depended on by this toolbox.
 
 ## What's New
 
-**2.25.0** was released in 1/6/2022:
+**v3.0.0rc0** was released in 31/8/2022:
 
-- Support dedicated `MMDetWandbHook` hook
-- Support [ConvNeXt](configs/convnext), [DDOD](configs/ddod), [SOLOv2](configs/solov2)
-- Support [Mask2Former](configs/mask2former) for instance segmentation
-- Rename [config files of Mask2Former](configs/mask2former)
-
-Please refer to [changelog.md](docs/en/changelog.md) for details and release history.
-
-For compatibility changes between different versions of MMDetection, please refer to [compatibility.md](docs/en/compatibility.md).
+- Faster and smoother training based on [MMEngine](https://github.com/open-mmlab/mmengine).
```

Comment:
- Unifies interfaces of all components based on MMEngine.
- Faster training and testing speed with complete support of mixed precision training.
- Provides more strong baselines and a general semi-supervised object detection framework. See [tutorial of semi-supervised detection](https://mmdetection.readthedocs.io/en/v3.0.0rc0/user_guides/semi_det.html).
- Allows any kind of single-stage model as an RPN in a two-stage model. See [tutorial](https://mmdetection.readthedocs.io/en/v3.0.0rc0/user_guides/single_stage_as_rpn.html).